Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './Footer.css';
+import reload from './img/reload.png';
+import speech from './img/speech-bubble.png';
+import heart from './img/like.png';
+import mail from './img/mail.png';
+
+const Footer = props => {
+    return (
+        <div className="footer">
+            <img src={speech} className="speech-bubble" />
+            <img src={reload} className="arrow-circle" /> <span>{props.retweet}</span>
```

Comment:
You're one step closer to mastering React, by using props, you're already a day ahead.